sys/mobileapps.yaml

```diff
@@ -74,13 +65,17 @@ paths:
         - check_storage_lead:
             request:
               method: get
-              uri: /{domain}/sys/key_value/mobileapps.lead/{title}/{$.request.body.items[0].tid}
+              uri: /{domain}/sys/key_value/mobileapps.lead/{title}
+              headers:
+                cache-control: '{cache-control}'
```

Comment:
There's no need to send the header to `key_value` here, as we will never get here with a `no-cache` value of that header.